dd-sdk-android-gradle-plugin/src/main/kotlin/com/datadog/gradle/plugin/internal/OkHttpUploader.kt

```diff
@@ -81,12 +88,16 @@ internal class OkHttpUploader : Uploader {
                     .format(mappingFile.absolutePath)
             )
         }
+
+        val eventJson = JSONObject()
+        eventJson.put("version", identifier.version)
+        eventJson.put("service", identifier.serviceName)
+        eventJson.put("variant", identifier.variant)
+        eventJson.put("type", TYPE_JVM_MAPPING_FILE)
+
         val builder = MultipartBody.Builder()
         builder.setType(MultipartBody.FORM)
-            .addFormDataPart("version", identifier.version)
-            .addFormDataPart("service", identifier.serviceName)
-            .addFormDataPart("variant", identifier.variant)
-            .addFormDataPart("type", TYPE_JVM_MAPPING_FILE)
+            .addFormDataPart("event", "event", eventJson.toString(0).toRequestBody(MEDIA_TYPE_JSON))
             .addFormDataPart("jvm_mapping_file", mappingFile.name, mappingFileBody)
```

Comment:
```suggestion
            .addFormDataPart("jvm_mapping_file", "jvm_mapping", mappingFileBody)
```
Could we set a consistent `jvm_mapping` file name here ? This is required by the v2 intake to correctly process the blob

Reply:
And same for the repository file, I think it needs `repository` as a filename (the v2 intake relies on file names instead of data part names)